src/components/chat/ChatContainer.js

```diff
@@ -5,25 +5,25 @@ import MessageForm from './MessageForm';
 import AlertContainer from '../general/AlertContainer';
 
 class ChatContainer extends Component {
-  render(){
+  render() {
     let { messages, username, onSendMessage } = this.props;
-    let { showAlert, alertMessage, alertStyle, onAlertDismiss } = this.props;
+    let { alertMessage, alertStyle, onAlertDismiss } = this.props;
 
     return (
       <div className="content">
 
         <AlertContainer
-            showAlert={showAlert}
-            message={alertMessage}
-            alertStyle={alertStyle}
-            onAlertDismiss={onAlertDismiss} />
+          message={alertMessage}
+          alertStyle={alertStyle}
+          onAlertDismiss={onAlertDismiss} />
 
         <MessageBox
           messages={messages}
           username={username} />
 
         <MessageForm
-          onSendMessage={onSendMessage} />
+          onSendMessage={onSendMessage}
+          hasFocus={this.props.messageInputFocus} />
```

Comment:
@Spetastian How about `shouldHaveFocus`?  I think that's what we're using this for, correct?